recipes/setup_server.rb

```diff
@@ -28,4 +28,17 @@
     notifies :restart, "service[apache2]"
 end
 
+template "/etc/init.d/websockify" do
+    source "websockify.erb"
+    owner "root"
+    group "root"
+    variables({
+        :websockify_port => node['abiquo']['websockify']['port'],
+        :websockify_key => node['abiquo']['websockify']['key'],
+        :websockify_crt => node['abiquo']['websockify']['crt']
+    })
```

Comment:
[hint] There's no need to pass the variables. You can omit the entire variables construct and directly access the node properties from the template file.